includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -1063,6 +1073,19 @@ class="<?php echo esc_attr( $field_options['class'] ); ?>"
 			</tr>
 			<?php
 		}
+
+		/**
+		 * @since 2.8.0
+		 */
+		public function disconnect_connected_stripe_accout() {
+			$data = give_clean( $_GET );
```

Comment:
@kjohnson Can we tighten security here:
- Add nonce validation
- Check for user permission (must have)

Reply:
User permission check resolved by commit 9d10699caa4c7ab0bba3a34ec208889377d8b930

Reply:
Nonce added with commit a812810353d0acce487aeaf65680ca43fe67131a

Reply:
Looks good.